cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -91,19 +103,33 @@ public void initScopedTransformersAndKeys() {
         return filter(having(on(Transformer.class).getScopes().contains(scope)), transformers);
     }
 
-    public Executable transformToExecutable(ParsedSlang parsedSlang, String execName, Map<String, Object> executableRawData) {
+    @SuppressWarnings("unchecked")
```

Comment:
I am not much of a fan of `SupressWarnings` on methods..

can we use some tricks? like:
```java
@SuppressWarnings("unchecked")
List<Map<String, Map<String, Object>>> onFailureDataHelper = (List<Map<String, Map<String, Object>>>)taskData.values().iterator().next();
onFailureData = onFailureDataHelper;
```